src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageAccountTests.ps1

```diff
@@ -910,11 +910,6 @@ function Test-FailoverAzureStorageAccount
         Assert-AreEqual $kind $sto.Kind; 
 		$seconcaryLocation = $sto.SecondaryLocation
 
-		# check Last Sync Time
-		$getLSTResult = Get-AzureRmStorageAccountLastSyncTime -ResourceGroupName $rgname -Name $stoname
```

Comment:
Please remove this cmdlet from the two psd1 files

Reply:
Please also add a changelog update.

Reply:
Removed from the 2 psd1 file. And change log updated. (change log already added for release for failover, but not released, so just update it)